### ISSUE TITLE
Request for new CV entries

### ISSUE THREAD

**Issue by rolivella** [HUPO-PSI/qcML-development#42]
Dear all, 

Julian told me that I can request new CV entries here. Ideally we would like to have all the metrics that we are taking into account in the QCloud (see Table 1 of our paper http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0189209). By order of priority, we would like to have the **Peak Area**, **Mass accuracy**, **Retention Time**, **Median Injection Time for MS2**, **Chromatographic Resolution**, **Peak Capacity** and **Total Ion Current**. 

Thank you! 

Roger

**Comment by rolivella** [comment]
Do you mean that I should use this MS CVs or are you going to add them into the QC CVs? 

Besides, I'm not 100% sure that these MS CVs are exactly the same parameters we are using. For instance, we're using the median IT of identified peptides in MS1 and MS2 (NIST codes MS1-1 and MS2-1). Is "MS:1000927: ion injection time" referring to exactly the same? Also instead of the "MS:1000285: total ion current" we are indeed computing the sum of all TICs instead of the TIC for one particular spectrum...

**Comment by julianu** [comment]
I will review them and think about, whether it is a good idea to use the entries of the MS CV.
For some it might work, but it might also be cleaner to make new terms with a "QC metric of" to the name or something.

**Comment by rolivella** [comment]
OK thank you Julian. I also want to let you know that we have a forked version of your main qc-cv.obo document with the QC parameters we are working on in the new version of the QCloud. We also created a **lc-cv.obo** (for Liquid Cromatographers) and **nc-cv.obo** (for non conformities reasons). We don't want to replace at all your work, for us your documents are the main reference, but we need something more operational to do not stop our work. If you want I can invite you to the Github where we have this files and you can have a look. As you wish, just let me know.

**Comment by julianu** [comment]
Hej roger,
Apologies for the long silence, we are "cleaning" the in-work-CV right now and during this I could add your proposed terms as well.
For some of them it is not clear to me, whether the actual values are averages or whether you report the values for each identified spectrum/peptide/feature, though. This is for the terms "Peak Area", "Mass Accuracy" and "Retention Time Drift": should they be tuples or single values? Or (what we also have by now) corresponding lists, where one list e.g. gives feature IDs and the other one the value?

All Other terms seem to be single values, right?

**Comment by rolivella** [comment]
Hi there! Thanks for your answer. In fact my previous message became old because we are using like a fork of your qcML doc beacause we need it for our QCloud v2 we are implementing. You can see it at: 

https://github.com/rolivella/cv/blob/master/qc-cv.obo

I'll give you access to it. Do you think that we can actually create a fork of your doc and eventually merge them in the future?

**Comment by rolivella** [comment]
Hi guys! We would like to

**Comment by mwalzer** [comment]
So a branch would always be an option, of course. We have the 'old' CV in one file and the restructured in another (qcML-development/cv/v0_0_11/). 
I think it would be best if you'd still in parallel open issues for each term you add (so we can follow up). I just created a label for QCloud-CV terms, you should add to the issue to let us know there is no *immediate* action item to be followed up on because you changed it in the branch already.
@julianu what do you think?

**Comment by julianu** [comment]
I think requesting to add them into the CV is the best option for now, yes. If you already branched it, we can of course merge your additions back into the current version as well.

**Comment by mwalzer** [comment]
As discussed lc.obo requested to go to PSI-MS CV,
let's have a look at the remainder

**Comment by mwalzer** [comment]
Maybe tackle the nc.obo next?

**Comment by rolivella** [comment]
As discussed with Mathias, I attach the work we've done with lc (liquid cromatography) and nc (non-conformities) controlled vocabulary:

[nc-cv.txt](https://github.com/HUPO-PSI/mzQC/files/2880948/nc-cv.txt)
[lc-cv.txt](https://github.com/HUPO-PSI/mzQC/files/2880949/lc-cv.txt)

**Comment by rolivella** [comment]
I copy&paste first the lc-cv:

[Term]
id: LC:0000001
name: Proxeon Biosystems instrument model
def: "Proxeon Biosystems instrument model."
is_a: MS:1000031 ! instrument model

[Term]
LC:0000002
name: EASY-nLC™ II LC 110
def: "Proxeon Biosystems Easy-nLC II LC110."
is_a: LC:0000001 ! Proxeon Biosystems instrument model

[Term]
LC:0000003
name: EASY-nLC™ 1000 LC 120
def: "Proxeon Biosystems Easy-nLC 1000 LC120."
is_a: LC:0000001 ! Proxeon Biosystems instrument model

[Term]
LC:0000004
name: Eksigent instrument model
def: "Eksigent instrument model."
is_a: MS:1000031 ! instrument model

[Term]
LC:0000005
name: Eksigent autosampler
def: "Eksigent autosampler."
is_a: LC:0000004 ! Eksigent instrument model

[Term]
LC:0000006
name: Eksigent NanoLC AS-2 autosampler
def: "Eksigent NanoLC AS-2 autosampler."
is_a: is_a: LC:0000005 ! Eksigent autosampler

[Term]
LC:0000007
name: Eksigent Nano Ultra LC 1D plus system
def: "Eksigent Nano Ultra LC 1D plus system"
is_a: LC:0000004 ! Eksigent instrument model

**Comment by rolivella** [comment]
An then the nc-cv: 

[Term]
id: NC:0000001
name: Non conformity
def: "Instrument indicia that are associated with a performance out of the limits of acceptance."
is_a: MS:1000547 ! Object attribute

[Term]
id: NC:0000002
name: Non conformity reason
def: "Cause that genrates a non conformity."
is_a: MS:1000547 ! Object attribute

[Term]
id: NC:0000003
name: Decrease of MS1 signal
def: "The intensity of the signal in MS1 is lower than expected"
is_a: NC:0000001 ! Non conformity

[Term]
id: NC:0000004
name: Sample carry over
def: "Part of the analyte present in a sample appears to be presest in the next or following samples."
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000003 ! Loss of MS1 signal & part_of NC:0000011 The intensity of the signal in MS1 fluctuates & part_of NC 0000019 ! No signal in MS1 & part_of NC 0000020 ! Bad chromatography

[Term]
id: NC:0000005
name: Air bubble in QC sample vial
def: "Air bubbles are present in the QC sample vial."
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000003 ! Loss of MS1 signal

[Term]
id: NC:0000006
name: Bad QC sample
def: "QC sample is not correct."
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000003 ! Loss of MS1 signal

[Term]
id: NC:0000007
name: Empty solvents
def: "Bottles from LC solvents are empty."
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000003 ! Loss of MS1 signal & part_of NC 0000019 ! No signal in MS1

[Term]
id: NC:0000008
name: Air in the chromatographic path
def: "Air bubbles in the chromatographic path."
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000003 ! Loss of MS1 signal &  part of NC:0000020 ! Bad chromatography

[Term]
id: NC:0000009
name: Deteriorated column
def: "Chromatographic column is deteriorated."
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000003 ! Loss of MS1 signal & part_of NC 0000014 ! Loss of MS1 signal in the central part of the chromatogram & part_of NC 0000020 ! Bad chromatography

[Term]
id: NC:0000010
name: Column overpressure
def: "The column pressure is above the set limits"
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000003 ! Loss of MS1 signal

[Term]
id: NC:0000011
name: MS1 signal fluctuation
def: "The intensity of the signal in MS1 is sometimes lower than expected and sometimes inside the thresholds of acceptance."
is_a: NC:0000001 ! Non conformity

[Term]
id: NC:0000012
name: Dirty column emitter
def: "The emitter of the column is dirty"
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000011 The intensity of the signal in MS1 fluctuates & part_of NC 0000019 ! No signal in MS1

[Term]
id: NC:0000013
name: Dirty ion transfer tube
def: "The ion transfer tube is dirty"
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000011 The intensity of the signal in MS1 fluctuates and part_of NC 0000019 ! No signal in MS1

[Term]
id: NC:0000014
name: Loss of MS1 signal in the central part of the chromatogram
def: "The intensity of the signal in MS1 for complex samples is lower than expected only in the central part of the chromatogram"
is_a: NC:0000001 ! Non conformity

[Term]
id: NC:0000015
name: Dirty ion path 
def: "The ion path is dirty"
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000014 Loss of MS1 signal in the central part of the chromatogram

[Term]
id: NC:0000016
name: Sample injection step does not finish 
def: "Sample injection process get stacked in the loading sample step"
is_a: NC:0000001 ! Non conformity

[Term]
id: NC:0000017
name: Sample too concentrated
def: "The amount of injected sample is too high"
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000016 Sample injection step does not finish

[Term]
id: NC:0000018
name: Dirty sample
def: "Sample is contaminated with non peptidic compounds"
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000016 Sample injection step does not finish

[Term]
id: NC:0000019
name: No signal in MS1
def: "There is no signal in MS1"
is_a: NC:0000001 ! Non conformity

[Term]
id: NC:0000020
name: Bad chromatography
def: "The shape, intensity or retention times of the chromatographic profile do not correspond with the expected"
is_a: NC:0000001 ! Non conformity

[Term]
id: NC:0000021
name: Leak in the chromatographic system
def: "There is a leak of solvent in the chromatographic system"
is_a: NC:0000002 ! Non conformity reason
part_of NC 0000020 ! Bad chromatography

[Term]
id: NC:0000022
name: Uncalibrated masses
def: "The mass accuracy of the measured peptides is out of specifications"
is_a: NC:0000001 ! Non conformity

[Term]
id: NC:0000023
name: Communication error
def: "The acquisition computer cannot connect with the instrument"
is_a: NC:0000001 ! Non conformity

**Comment by rsalek** [comment]
Can I make a suggestion to make it more generic, hence fitting metabolomics too?
e.g. 
[Term]
id: NC:0000018
name: Dirty sample
def: "Sample is contaminated with **_undesirable_** compounds"
is_a: NC:0000002 ! Non conformity reason
part_of NC:0000016 Sample injection step does not finish. <---- Not sure it is related to? Just will creates a poor quality spectrum!


[Term]
id: NC:0000022
name: Uncalibrated masses
def: "The mass accuracy of the measured **_molecules_** is out of specifications"
is_a: NC:0000001 ! Non conformity

**Comment by mwalzer** [comment]
:+1:

**Comment by rsalek** [comment]
Do we have CV entries for, internal standard -different types-, system suitability and other types of QC present in a run?

**Comment by julianu** [comment]
I have urgently to go through the CV, but as far as I remember we don't have them yet.
But feel free to ask for them (maybe with some small sample), and best in a new issue, which we will tag as CV request.
